src/main/scala/org/broadinstitute/dsde/firecloud/model/ElasticSearch.scala

```diff
@@ -11,6 +11,9 @@ object ElasticSearch {
   final val fieldAll = "_all"
   final val fieldSuggest = "_suggest"
   final val fieldDiscoverableByGroups = "_discoverableByGroups"
+  final val fieldOntologyParents = "parents"
+  final val fieldOntologyParentsOrder = "order"
+  final val fieldOntologyParentsLabel = "label"
```

Comment:
Curious about why we're not using underscores like we are with `_all`, `_suggest`, etc.